src/ManagerV2/FolderManager.php

```diff
@@ -37,25 +35,15 @@ public function move(Dossier $folder, ?Dossier $parentFolder): void
         } elseif ($parentFolder !== $folder) {
             $parentFolder->addSousDossier($folder);
             $folder->setBPrive($parentFolder->getBPrive() || $folder->getBprive());
-            $this->toggleVisibility($folder, $folder->getBPrive());
+            $this->toggleVisibility($folder);
```

Comment:
There is something weird here, we set $folder BPrive and then we toggle its visibility